src/Tribe/Attendee_Registration/Main.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Attendee Registration core class
+ *
+ * @since TBD
+ */
+class Tribe__Tickets__Attendee_Registration__Main {
+	const QUERY_VAR = 'attendee-registration';
+	const DEFAULT_PAGE_SLUG = 'attendee-registration';
+
+	/**
+	 * Retrieve the attendee registration slug
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	public function get_slug() {
+		return Tribe__Settings_Manager::get_option( 'ticket-attendee-info-slug', self::DEFAULT_PAGE_SLUG );
+	}
+
+	/**
+	 * Returns whether or not the user is on the attendee registration page
+	 *
+	 * @since TBD
+	 *
+	 * @return bool
+	 */
+	public function is_on_page() {
+		global $wp_query;
+
+		return ! empty( $wp_query->query_vars[ self::QUERY_VAR ] );
+	}
+
+	/**
+	 * Gets the URL for the attendee registration page
+	 *
+	 * @return string
+	 */
+	public function get_url() {
+		$slug = $this->get_slug();
+
+		return home_url( "/{$slug}/" );
+	}
+
+	public function get_checkout_url() {
```

Comment:
missing docblock

Reply:
You can pretty much copy the one below (for the filter), then change that one (for the filter) to something along the lines of `Allows filtering of the checkout url` or some such.

Reply:
Thanks 👍